docs/reference/components/sources/aws_kinesis_firehose.cue

```diff
@@ -1,6 +1,6 @@
 package metadata
 
-components: sources: aws_kinesis_firehose: components._aws & {
+components: sources: aws_kinesis_firehose: {
```

Comment:
Is this intended?

Reply:
Yes. `components._aws` adds options and documentations that aren't applicable to `aws_kinesis_firehose` source.